docs/source/installation.rst

```diff
@@ -0,0 +1,50 @@
+.. _installation:
+
+Installation
+============
+It is highly recommended to install the project into a new virtual environment.
+
+
+#Installation from PyPI
+#----------------------
+#The :code:`atldld` package can be easily installed from PyPI.
+#
+#
+#.. code-block:: bash
+#
+#    pip install atldld
+
+Installation from source
+------------------------
+As an alternative to installing from PyPI, if you want to try the latest version
+you can also install from source.
+
+.. code-block:: bash
+
+    pip install git+https://github.com/BlueBrain/Atlas-Download-Tools
+
+Development installation
+------------------------
+For development installation one needs additional dependencies grouped in :code:`extras_requires` in the
+following way:
+
+- **dev** - pytest + plugins, flake8, pydocstyle, tox
+- **docs** - sphinx
+
+.. code-block:: bash
+
+    git clone https://github.com/BlueBrain/Atlas-Download-Tools
+    cd atlas_alignment
+    pip install -e .[dev,docs]
+
+
+Generating documentation
+------------------------
+To generate the documentation make sure you have dependencies from :code:`extras_requires` - :code:`docs`.
+
+.. code-block:: bash
+
+    cd docs
+    make clean && make html
+
+One can view the docs by opening :code:`docs/_build/html/index.html` in a browser.
```

Comment:
Maybe we should also document somewhere how to re-generated the API docs - do you still have the command?

Maybe here is not the right place though since re-generating the API docs is for devs and not for users.

Reply:
I agree it could be useful to have this document somewhere. However, I am not sure about the right place for this!

Reply:
Maybe `CONTRIBUTING.md` could be an option?

Reply:
Hello Stan, I tried to integrate your comment in a1500b8. 
Just for you to know, initially, I changed slightly the structure of the documentation. Do not hesitate to compare the previous version (that I changed slightly) and the new one (completely generated thanks to `sphinx-apidoc`) and tell me what you prefer!

Reply:
I think I prefer when each submodule has its own page. (So the old layout?) How about you?

I wasn't actually able to reproduce the old structure, do you know how Jan generated it initially?

I can get a similar one, but not exactly the same with
```bash
sphinx-apidoc src/atldld -Tefo docs/source/api
```
after that one needs to remove the `api/module.rst` file since it's not used any more and to edit `index.rst` and replace `source/api/modules` by `source/api/atldld`. Then
```bash
make -C docs clean html
```
You can try if you want and see what you think about this one.

Reply:
It might be a good compromise! I also prefer having each module on a different page. Do you agree if I commit this version ?

Reply:
Sure, if you like it we can go for it :)

Reply:
Done in 867246e.

Reply:
awesome, thanks!